<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Background{
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.21, x2:0.5, y2:0.89, stop:0.0105263 rgba(58, 72, 140, 237), stop:1 rgba(83, 125, 255, 237));
}</string>
   </property>
   <widget class="QWidget" name="Background_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>520</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#Background{
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.21, x2:0.5, y2:0.89, stop:0.0105263 rgba(58, 72, 140, 237), stop:1 rgba(83, 125, 255, 237));
}</string>
    </property>
    <widget class="QLineEdit" name="UserUP">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>180</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: #D9ED92;
font: 17pt &quot;Arial&quot;;
border: 5px dashed Transparent;
border-bottom-color: #00241b;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Cédula</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PassUP">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>260</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: #D9ED92;
font: 17pt &quot;Arial&quot;;
border: 5px dashed Transparent;
border-bottom-color: #00241b;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Contraseña</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Sign_BtnBD">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>340</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#Sign_BtnBD{
background-color: #006BA6;
border-radius: 10px;
font: bold 14pt &quot;Arial&quot;;
color: #FCFFE7;
}

QPushButton#Sign_BtnBD:pressed{
background-color: #3E6D9C;
color: #F0F0F0;
}</string>
     </property>
     <property name="text">
      <string>Registrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Return_Btn">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>340</y>
       <width>170</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#Return_Btn{
background-color: #ffdb57;
border-radius: 10px;
font: bold 14pt &quot;Arial&quot;;
color: #00000;
}

QPushButton#Return_Btn:pressed{
background-color: #3E6D9C;
color: #F0F0F0;
}</string>
     </property>
     <property name="text">
      <string>Volver a
Inicio de Sesión</string>
     </property>
    </widget>
    <widget class="QLabel" name="Bienv_title_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>581</width>
       <height>51</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 35pt &quot;Arial Black&quot;;
color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Bienvenido!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="Bienv_title">
     <property name="geometry">
      <rect>
       <x>424</x>
       <y>100</y>
       <width>581</width>
       <height>51</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 bold 22pt &quot;Arial Black&quot;;
color: #b7e4c7;</string>
     </property>
     <property name="text">
      <string>Registro de Usuario</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>440</y>
       <width>501</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;
color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Autor: Br. Laura Rodriguez
Ingeniería de Sistemas
I-2023</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1000</width>
       <height>520</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;/home/Laura/Escritorio/SISTEMA 1.0/icons/back1000520.png&quot; width=&quot;1000&quot; height=&quot;520&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>UserUP</zorder>
    <zorder>PassUP</zorder>
    <zorder>Sign_BtnBD</zorder>
    <zorder>Return_Btn</zorder>
    <zorder>Bienv_title_2</zorder>
    <zorder>Bienv_title</zorder>
    <zorder>label</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
